test/spring/balance/balance_test.exs

```diff
@@ -0,0 +1,153 @@
+defmodule Spring.Balance.BalanceTest do
+  use ExUnit.Case, async: true
+  use Hailstorm.SpringHelper
+  require Logger
+
+  test "default balancer" do
+    host_name = "balancer_host"
+    username = "PlayfulRoseRecluse"
+
+    host_socket =
+      new_connection(%{
+        name: host_name,
+        roles: ["Bot"],
+        update: %{
+          bot: true
+        }
+      })
+
+    user_socket =
+      new_connection(%{
+        name: username
+      })
+
+    # TeiServer will create a user but change the name by adding _hailstorm
+    username = username <> "_hailstorm"
+
+    lobby_id = Commands.open_lobby(host_socket, host_name, name: "lobby name")
+
+    Commands.join_lobby(user_socket, username, host_socket, lobby_id)
+
+    # This changes the user to be in team 1
+    # Search TeiServer repo for that weird number 4195330
+    # Is there a better way to add a user to a team?
```

Comment:
https://springrts.com/dl/LobbyProtocol/ProtocolDescription.html
Add link to spring protocol in comments